### ISSUE TITLE
Error on Install

### ISSUE THREAD

**Issue by ridgeandgable** [nabeelio/phpvms#292]
I've downloaded the Version 7 Alpha 2 , put the downloaded file into my HTDOCS (Xampp), went to CMD and ran Composer Install.. All went fine, loaded the site, localhost/phpvms
I get the message, not installed, I click on the green button and everything is checked OK and green, goto Database setup, enter all my details, I do check database (phpvms) and it says ok, I click on next or install, and then get an error

`
C:\xampp\htdocs\phpvms\vendor\laravel\framework\src\Illuminate\Database\Connection.php

 * Execute an SQL statement and return the boolean result. * * @param string $query * @param array $bindings * @return bool */ public function statement($query, $bindings = []) { return $this->run($query, $bindings, function ($query, $bindings) { if ($this->pretending()) { return true; } $statement = $this->getPdo()->prepare($query); $this->bindValues($statement, $this->prepareBindings($bindings)); $this->recordsHaveBeenModified(); return $statement->execute(); }); } /** * Run an SQL statement and get the number of rows affected. * * @param string $query * @param array $bindings * @return int */ public function affectingStatement($query, $bindings = []) { return $this->run($query, $bindings, function ($query, $bindings) { if ($this->pretending()) { return 0; } // For update or delete statements, we want to get the number of rows affected // by the statement and return that back to the developer. We'll first need // to execute the statement and then we'll use PDO to fetch the affected. Arguments "Maximum execution time of 30 seconds exceeded" Environment & details: GET Data empty POST Data empty Files empty Cookies phpvms_session "eyJpdiI6Imk2QjRMNGRzZER3SlVmcDJ2aURDd0E9PSIsInZhbHVlIjoib0hcL296cWlpUEtsbjFsb1F3N1NCR2J2aDJlR3pOcXFMa1lidFBHR1p2TDJ2XC9lNklDSGppZFZWSVJnVWtwelRXV3pxR0U2UmR4UE9I ▶" Session empty Server/Request Data REDIRECT_HTTP_AUTHORIZATION "" REDIRECT_MIBDIRS "C:/xampp/php/extras/mibs" REDIRECT_MYSQL_HOME "\xampp\mysql\bin" REDIRECT_OPENSSL_CONF "C:/xampp/apache/bin/openssl.cnf" REDIRECT_PHP_PEAR_SYSCONF_DIR "\xampp\php" REDIRECT_PHPRC "\xampp\php" REDIRECT_TMP "\xampp\tmp" REDIRECT_STATUS "200" HTTP_AUTHORIZATION "" MIBDIRS "C:/xampp/php/extras/mibs" MYSQL_HOME "\xampp\mysql\bin" OPENSSL_CONF "C:/xampp/apache/bin/openssl.cnf" PHP_PEAR_SYSCONF_DIR "\xampp\php" PHPRC "\xampp\php" TMP "\xampp\tmp" HTTP_HOST "localhost" HTTP_CONNECTION "keep-alive" HTTP_CACHE_CONTROL "max-age=0" HTTP_UPGRADE_INSECURE_REQUESTS "1" HTTP_USER_AGENT "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36" HTTP_ACCEPT "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8" HTTP_REFERER "http://localhost/phpvms/install/step2?" HTTP_ACCEPT_ENCODING "gzip, deflate, br" HTTP_ACCEPT_LANGUAGE "en-GB,en-US;q=0.9,en;q=0.8" HTTP_COOKIE "phpvms_session=eyJpdiI6Imk2QjRMNGRzZER3SlVmcDJ2aURDd0E9PSIsInZhbHVlIjoib0hcL296cWlpUEtsbjFsb1F3N1NCR2J2aDJlR3pOcXFMa1lidFBHR1p2TDJ2XC9lNklDSGppZFZWSVJnVWtwelRXV ▶" PATH "C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Common Files\Research Machines\;C:\Pr ▶" SystemRoot "C:\WINDOWS" COMSPEC "C:\WINDOWS\system32\cmd.exe" PATHEXT ".COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC" WINDIR "C:\WINDOWS" SERVER_SIGNATURE "<address>Apache/2.4.37 (Win32) OpenSSL/1.1.1a PHP/7.3.0 Server at localhost Port 80</address>\n" SERVER_SOFTWARE "Apache/2.4.37 (Win32) OpenSSL/1.1.1a PHP/7.3.0" SERVER_NAME "localhost" SERVER_ADDR "::1" SERVER_PORT "80" REMOTE_ADDR "::1" DOCUMENT_ROOT "C:/xampp/htdocs" REQUEST_SCHEME "http" CONTEXT_PREFIX "" CONTEXT_DOCUMENT_ROOT "C:/xampp/htdocs" SERVER_ADMIN "postmaster@localhost" SCRIPT_FILENAME "C:/xampp/htdocs/phpvms/index.php" REMOTE_PORT "63544" REDIRECT_URL "/phpvms/install/dbsetup" GATEWAY_INTERFACE "CGI/1.1" SERVER_PROTOCOL "HTTP/1.1" REQUEST_METHOD "GET" QUERY_STRING "" REQUEST_URI "/phpvms/install/dbsetup" SCRIPT_NAME "/phpvms/index.php" PHP_SELF "/phpvms/index.php" REQUEST_TIME_FLOAT 1547475354.353 REQUEST_TIME 1547475354 APP_KEY "base64:Mq/5ZUgmfRY9i9C4ndxj2TSuRYgRPAN+uwpxG36Rtm0=" SHELL_VERBOSITY 0 Environment Variables APP_KEY "base64:Mq/5ZUgmfRY9i9C4ndxj2TSuRYgRPAN+uwpxG36Rtm0=" SHELL_VERBOSITY 0 Registered Handlers 0. Whoops\Handler\PrettyPageHandler 

`

It won't let me do anything else?

**Comment by nabeelio** [comment]
Can you surround the message with ``` or include a screenshot? It's a bit
hard to read. I haven't tested it through xampp

>

**Comment by ridgeandgable** [comment]
Update, it didn't work either. I get the Error 500 on install. Version 7 Alpha 1

More than happy to give someone my details to help get it working :D

**Comment by nabeelio** [comment]
Strange. It could be something to do with htaccess and URL rewrites. If you
can install from scratch, and then attach the logs from storage/logs, make
sure there's no sensitive data in there first. That'll go a way in trying
to debug and diagnose

>

**Comment by ridgeandgable** [comment]
No bother will give it a go

I've got version 7 running on campus, but getting on homepage "trying to get property of non-object"

If Campo is the only way to do it then I guess I'll settle for that. Will post back later.

Thanks

⁣Sent from BlueMail ​

**Comment by nabeelio** [comment]
Well it would still be helpful to know and fix it elsewhere - I'm sure
someone else will run into the issue as well so it'd be good to make sure
those cases are covered

>

**Comment by ridgeandgable** [comment]
Just wiping out everything on the server now. What version do you suggest I try?

I ran a PHP info test to see if 1and1 have something turned off, but they meet everything required. It must be a htaccess fault somewhere. I can't find a publichtml folder anywhere.

I've tried in main directory (root) and I've also created a folder named phpvms, still errors

I've tried alpha 1 and 2 and even beta versions. I've tested on 5.6, 7, 7.1, 7.2 and still the same 

I'll wipe out everything on 1and1 and clean install again with alpha 2. Is there a copy somewhere that doesn't need the composer? I've tried .tar and the source from the GitHub site.

Cheers

**Comment by ridgeandgable** [comment]
***UPDATE***

after a lot of playing around, I managed to get it working.. It seems it doesn't like to be named in the root folder other than phpvms

I also noticed on several test on my system, that if I quickly went through the setup and didn't change the VA name on DB setup, it would fail with a timeout 30 error

What did work for me (1and1 hosted) was, to composer Update, then composer install
create the folder phpvms on root, name the VA in DB setup to anything other than phpvms and everything else went smoothly

I've spent the last 5 hours trying different things, and that's what work... Also, in the root / of 1and1, needed a htaccess file with RewriteBase /phpvms/ as did the htaccess in main folder

**Comment by ridgeandgable** [comment]
Only error I have found is on Add airport... If I use lookup for EGPH, the coordinates highlight red and won't let me save... Seems if I shorten the co-ords down to about 8 digits, all is good... Is there a fix for this?

**Comment by nabeelio** [comment]
So I don't quite get the 1and1 issue, are you on a VPS? Or do they rehome `/` to point to your own directory?

Also with the coords, the DB field is limited to 8 digits. You don't really need more than that. This is a great explanation about the accuracy and precision required:
https://gis.stackexchange.com/a/8674

In this case, you don't really need much more than 5 decimal places, maybe 6.

**Comment by ridgeandgable** [comment]
Hi Nabeelio

I have xampp locally for testing, but I have a hosted package with 1 and 1, and that's the one that's been giving me the issue, I saw there was a few people in the forum with Godaddy and 1and1 who were having issues to, hence my full explanation above as to what worked for me, as this may help them

Version 7 is amazing, so happy that I kept fighting to get this to work :)

The issue with Coords isn't a big deal

I assume the setup for FS Flight Keeper is still the same as Version 5?

Whats happening with main Support forum on website. I get the message, Banned.. Have I said something wrong?

Thanks for your help and your great software!
